### PR TITLE
Update buttons and add popup windows for them in Battle Only mode

### PR DESCRIPTION
**Pull request by ihhub**
relates to #4352

~~Also fixed enumeration entries misspelling~~

image

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @zenseii , can you please review newly added text?

File: src/fheroes2/agg/agg_image.cpp [hunk 1]
```diff
@@ -105,8 +105,8 @@ namespace
                                                 ICN::BUTTON_SMALL_EXIT_EVIL,
                                                 ICN::BUTTON_EXIT_HEROES_MEETING,
                                                 ICN::BUTTON_EXIT_TOWN,
-                                                ICN::BUTTON_EXIT_PUZZLE_DDOOR_GOOD,
-                                                ICN::BUTTON_EXIT_PUZZLE_DDOOR_EVIL,
+                                                ICN::BUTTON_EXIT_PUZZLE_DOOR_GOOD,
```
[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/agg/agg_image.cpp:108]: Ah, then I should revert changes, Thanks for pointing out!

File: src/fheroes2/battle/battle_only.cpp [hunk 2]
```diff
@@ -236,6 +236,16 @@ bool Battle::Only::setup( const bool allowBackup, bool & reset )
             break;
         }
 
+        if ( le.MousePressRight( buttonStart.area() ) ) {
+            fheroes2::showStandardTextMessage( _( "Start" ), _( "Start the battle." ), 0 );
+        }
+        else if ( le.MousePressRight( buttonCancel.area() ) ) {
+            fheroes2::showStandardTextMessage( _( "Cancel" ), _( "Exit this menu." ), 0 );
```
[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/battle/battle_only.cpp:243]: Well, we have a problem with generating EXIT button with shadow but yeah, I will do it.

[the remark this answers is not in the document]
Comment (Districh-ru) [reply src/fheroes2/battle/battle_only.cpp]: Hi, @zenseii, I've changed the CANCEL button to EXIT. This button will also adopt its size to the text.
изображение
изображение

File: src/fheroes2/agg/agg_image.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (ihhub) [reply src/fheroes2/agg/agg_image.cpp]: I renamed them from DDOOR to DIM_DOOR to be obvious.

**Review by Districh-ru — APPROVED** [review]

**Pull request merged**